package.json

```diff
@@ -83,6 +86,9 @@
     "warning": "^3.0.0"
   },
   "peerDependencies": {
+    "@gandi/babel-preset-gettext": "*",
+    "babel-cli": "*",
```

Comment:
You can add `babel-cli` as a `devDependencies`.

peerDependencies are useful for big libs that end in the build. Here, we don't really care.

BTW you can remove `@gandi/babel-preset-gettext` ;)